hrqb/tasks/libhr_employee_appointments.py

```diff
@@ -86,7 +110,7 @@ class LoadLibHREmployeeAppointments(QuickbaseUpsertTask):
     @property
     def merge_field(self) -> str | None:
         """Explicitly merge on unique Position ID field."""
-        return "Position ID"
+        return "Key"
```

Comment:
This indicates that, when upserting to Quickbase, we are updating rows based on the `Key` field, and no longer the `Position ID` field.